packages/client/src/app/shared/auth/services/auth.service.ts

```diff
@@ -0,0 +1,80 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, throwError } from 'rxjs';
+import { map, catchError } from 'rxjs/operators';
+
+import { FrontendUser, CreateUser, LoginUser } from '@pulp-fiction/models/users';
+
+/**
+ * ## AuthService
+ * 
+ * Manages API calls and processes return values for user authentication.
+ */
+@Injectable({
+    providedIn: 'root'
+})
+export class AuthService {
+    private url: string = `/api/auth`;
+
+    constructor(private http: HttpClient) {}
+
+    /**
+     * Sends a new user's info to the backend. If the response is successful,
+     * the `user` is returned and their info is set in local storage. Otherwise,
+     * sends the backend error to all subscribed observables.
```

Comment:
Is the bit about setting the info in local storage still true?

Reply:
yes--`ngxs/storage-plugin` automatically takes care of setting things in `localStorage`